language/move-vm/runtime/src/interpreter.rs

```diff
@@ -45,6 +45,12 @@ macro_rules! debug_writeln {
     };
 }
 
+macro_rules! set_err_info {
+    ($frame:ident, $e:expr) => {{
+        $e.at_code_offset($frame.function.index(), $frame.pc).finish($frame.location())
```

Comment:
Looks good, lets look at the other uses of set_location before landing

Reply:
Had a look at the other uses of `set_location` and they all seem correct to me. Additionally any issues happening in the main interpreter loop for a frame have the correct location since it's being derived from the frame that's being executed.